### ISSUE TITLE
[BUG] Parsons puzzle: second exercise is unreachable

### ISSUE THREAD

**Issue by MarleenGilsing** [hedyorg/hedy#4441]
![image](https://github.com/hedyorg/hedy/assets/80678586/31cb5891-f6c7-463c-9148-04e36998390e)
I can't click the second exercise and there seems to be no other button to get me to the second exercise?

**Comment by yilmazdurmaz** [comment]
I have found there is a "next parson" button coded, but does not seems linked in the code to activate, or the linked code has typos. I manually made the button visible from developer tools (disable "display:none;" style), and the button finely switches to the next parson.

https://github.com/hedyorg/hedy/blob/53e97d2d2ec3f43fb266d24de4439884b381375f/templates/incl/editor-and-output.html#L179

The header tabs are coded in this template. These also would benefit a rework to briefly show there are 2 or more parson. currently "Excercise" prefix moves when parson is changed only leaving the number. (I used above method to see this). a different color would also be better for them.

https://github.com/hedyorg/hedy/blob/53e97d2d2ec3f43fb266d24de4439884b381375f/templates/incl/parsons.html#L4-L10
